client/src/app/components/managing-body/managing-body.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="managingBody" (click)="onClick()">
+    <div class="">
```

Comment:
@tarieli1 Basically we don't add code that doesn't do anything right now. When we'll need that class we'll add them, but not an empty ones for future use.

But, you can leave it like that for now because this PR (Pull Request) will be merge after you'll finish with the `Semantic UI` one ( #34 ). So you'll add css to that PR and you'll probably need some of those classes.